modules/http4s-client/src/main/scala/io/janstenpickle/trace4cats/http4s/client/Http4sClientRequest.scala

```diff
@@ -6,8 +6,15 @@ import io.janstenpickle.trace4cats.model.{AttributeValue, SemanticAttributeKeys}
 import org.http4s.Uri
 
 object Http4sClientRequest {
+  // FIXME: move to trace4cats-model...
+  val SemanticAttributeKeys_httpFlavor: String = "http.flavor"
```

Comment:
I would be happy to move this into trace4cats-model if/once these changes are accepted.

Reply:
preemptively created PR for it: https://github.com/trace4cats/trace4cats/pull/659